src/graphql/schema/connection/createConnectionGqlField.ts

```diff
@@ -141,6 +142,7 @@ export function _createConnectionGqlType <TInput, TItemValue>(
   return new GraphQLObjectType({
     name: formatName.type(`${paginator.name}-connection`),
     description: `A connection to a list of ${scrib.type(gqlType)} values.`,
+    relatedGqlType: gqlType,
```

Comment:
Gotcha. That makes sense 👍

Maybe another name or some documentation on this name could be helpful, though.